apps/studio/data/tables/tables-query.ts

```diff
@@ -51,19 +64,19 @@ export async function getTables(
   if (Array.isArray(data) && sortByProperty) {
     return sortBy(data, (t) => t[sortByProperty]) as PostgresTable[]
   }
-  return data as PostgresTable[]
+  return data as Omit<PostgresTable, 'columns'>[]
```

Comment:
Columns will be included if `include_columns` is true though right? So wondering if using the `Omit` is correct?

Reply:
It returns no columns by default, so this type would be correct in most cases. When using `include_columns`, we would have to cast the return type ([example](https://github.com/supabase/supabase/pull/20157/files#diff-99d7ec12a697697fb966a710290a9603843572d25db8531694eccfce4090487fR67)). 

I don't have a strong opinion on this, just trying to make the default case more type-safe.

Reply:
I think casting is a viable solution for now – I don't think it's worth the time investment to get something "slightly" nicer 👍